client/src/components/menu/Menu.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import MenuListItem from "../menuListItem/MenuListItem";
+import PropTypes from "prop-types";
+import timeIconSrc from "../../assets/time.svg";
+import tasksIconSrc from "../../assets/tasks.svg";
+import euroIconSrc from "../../assets/euro.svg";
+import peopleIconSrc from "../../assets/people.svg";
+import MenuList from "../MenuList/MenuList";
+
+const Menu = ({ open }) => {
+  return (
+    <MenuList open={open}>
+      <MenuListItem to="/" text="Home" icon={timeIconSrc} />
```

Comment:
Maybe could rename it like following for a better code understanding?
```suggestion
      <MenuListItem linkDirection (or href)="/" text="Home" icon={timeIconSrc} />
```